library/core/registry/src/main/java/org/quiltmc/qsl/registry/impl/sync/ServerPackets.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.registry.impl.sync;
+
+import it.unimi.dsi.fastutil.ints.IntSet;
+import net.minecraft.util.Identifier;
+import org.jetbrains.annotations.ApiStatus;
+
+/**
+ * Identifiers of packets send by server
+ */
+@ApiStatus.Internal
+public final class ServerPackets {
```

Comment:
Are we sure we want to keep the registry sync packet structure an impl detail, instead of considering it part of the public API?

Reply:
Well structure could be part of public, but ideally mods shouldn't need it anyway. I guess documenting it somewhere is something we could look